.gitignore

```diff
@@ -67,6 +67,35 @@ doc/man1/openssl-verify.pod
 doc/man1/openssl-x509.pod
 doc/man1/openssl.pod
 
+# Auto generated doc files
```

Comment:
Could you please merge these with the list above as there are mostly duplicates?

Reply:
oops, done.  commit pushed.